test/components/form-attachment/list.spec.js

```diff
@@ -45,13 +45,17 @@ describe('FormAttachmentList', () => {
       }
     });
 
-    it('adds a title attribute for the attachment name', async () => {
-      testData.standardFormAttachments.createPast(1, { name: 'foo.jpg' });
+    it('shows attachment name if attachment does not exist', async () => {
+      testData.standardFormAttachments.createPast(1, {
+        name: 'foo.jpg',
+        blobExists: false
+      });
       const component = await load('/projects/1/forms/f/draft/attachments', {
         root: false
       });
-      const td = component.get('td.form-attachment-list-name');
-      td.attributes().title.should.equal('foo.jpg');
+      const span = component.get('td.form-attachment-list-name span');
+      span.text().should.equal('foo.jpg');
+      await span.should.have.textTooltip();
     });
```

Comment:
This is a new test. I realized that we didn't test the case where the Name column renders just text, not a link. In both cases, there should be a tooltip.